src/calculation/ClauseResultsBuilder.ts

```diff
@@ -736,7 +753,14 @@ export function buildPopulationRelevanceMap(
 
   // If DENEX is true then NUMER, NUMEX and DENEXCEP not calculated
   if (hasResult(PopulationType.DENEX, results) && getResult(PopulationType.DENEX, results) === true) {
-    if (!(group && MeasureBundleHelpers.hasMultipleIPPs(group))) {
+    // Do not apply this to ratio measures (numerator and denominator are independent from each other)
+    if (
+      !(
+        group &&
+        (MeasureBundleHelpers.getScoringCodeFromGroup(group) === MeasureScoreType.RATIO ||
+          measureScoringCode === MeasureScoreType.RATIO)
+      )
+    ) {
```

Comment:
I am having a hard time with this logic- 

We don't want to go into this if statement if it's a ratio measure, right?

If group doesn't exist but measureScoringCode === MeasureScoreType.RATIO, then the && calculates to false which then !false = true so it _does_ go into the if statement even though I don't think we want to since it's a ratio measure? Am I doing this logic correctly?
If group exists and its a ratio measure, then